Creation of the minutes from the 2022-12-02 meeting

Cristrinaranjus opened the pull request.

Cristrinaranjus committed:
first draft of 02-12-2022 minutes

Cristrinaranjus committed:
first draft of 02-12-2022 minutes

Cristrinaranjus committed:
Merge branch 'documentation/2022-12-02-minutes' of https://github.com/CCOMS-UAM/yos-award into documentation/2022-12-02-minutes

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+**Scribe:** Cristina Rodríguez
+
+**Schedule:** \<11:00\> - \<11:31\> <!-- # Init time - End time -->
```
Delete wildcard `<!-- # Init time - End time -->`

DaniMori replied:
Fill in "wildcard" **without** the <> signs (e.g. `11:00 - 11:31` instead of `<11:00> - <11:31>`)

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+---
+
+**Attendees:** Lea Francia, Roberto Mediavilla, Daniel Morillo, Cristina
+Rodríguez, Jesús Sierralaya
```
Check out @JesusSierralaya's format. We may want to decide on a standard format (enumeration separated by commas, or bullet points).

JesusSierralaya replied:
in this context I prefer my name just Jesus (without accent)

DaniMori replied:
@Cristrinaranjus mind @JesusSierralaya comment

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+## Approval of the previous minutes
+
+We left pending for the next meeting to approve the minutes of the first meeting
```
Use format in template:

```
#### Agreements

- **Postponed**: Approve minutes...
```

DaniMori replied:
The structure must be as follows:

```
# Action points

## Approval of minutes from 30/11/2022 meeting

#### Agreements

-   **Postponed**: *Approve of the previous minutes*. We left pending for the
    next meeting to approve the minutes of the first meeting taken by Jesus due
    to github usability issues.
```

Please mind the heading levels: each "action point" has/may have a "Debate", "Tasks" and "Agreements" section

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+taken by Jesus due to github usability issues.
+
+## Use of 'sourcetree' for manipulating branches on Git
+
```
Add header `#### Debate` as in template

DaniMori replied:
Add header `#### Debate` **inside** the action point `## Use of 'sourcetree' for manipulating branches on Git`, as in template

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+[here](https://www.sourcetreeapp.com/).
+
+## Use of canonical mode
+
```
Add header `#### Debate` as in template

DaniMori replied:
Add header `#### Debate` **inside** the action point `## Use of canonical mode`, as in template

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+We discuss how version history is recorded and who is changing what. A good tool
+to manage branches is 'sourcetree', which can be easily downloaded from
+[here](https://www.sourcetreeapp.com/).
```
Use bullet points as in template; this is intentional to try to keep the discussion topic concise.

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+The use of the canonical mode saves us conflicts when working with people who
+prefer to work with different modes (Source or Visual). This is because it
+establishes conventions that are always applied in Visual to Source mode. In our
+project it is set up to get as few headaches as possible in this matter.
```
Use bullet points as in template

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+project it is set up to get as few headaches as possible in this matter.
+
+## Skeleton of the proposal
+
```
Add header `#### Debate` as in template

DaniMori replied:
Add header `#### Debate` **inside** the action point `## Skeleton of the proposal`, as in template

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+After discussing different usability problems with Git and solving the issue of
+the branches to work on, Roberto presents us how he has structured the skeleton
+of the proposal. In this sense, it has been based on three key concepts:
```
Use bullet points as in template

DaniMori replied:
To keep it concise, do not use narrative of "what happened". Do not use names to say "who said what"; just state the "what"

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+To work in each of these sections, each one of us will create a branch that
+comes out of the branch `template-fillin` and that corresponds to our section.
+For example, in the case of Cris it would be `template-fillin/reusability`
```
State as "approved agreement" if necessary (see template for format)

DaniMori replied:
I don't see changes here. What I suggested:

```
#### Agreements

- **Approved**: bifurcaste branch `drafting/template-fillin` to fill in the corresponding section; e.g., `drafting/reusability` to fill in section "Reusability"

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+comes out of the branch `template-fillin` and that corresponds to our section.
+For example, in the case of Cris it would be `template-fillin/reusability`
+
+#### Tasks
```
I think the action point here is "Proposal fill in" or something (with `#### Tasks` subheader correctly nested)

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+#### Tasks
+
+-   \Lea Francia\>:
```
Watch out for "special character"; use `Lea Francia` only instead of `\Lea Francia\>` (same for the other points).

DaniMori replied:
Also, put member name in bold as in template, e.g. `**Lea Francia**`

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+
+-   \Lea Francia\>:
+
+    Outputs: What has the initiative achieved so far?
```
State the task as a verb to make it clearer: e.g. `Fill in field "Outputs: What has the initiative achieved so far?"`

Also, do not separate member and task (write where the wildcard is in the template)

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+# Suggestions, questions
+
+The Teams channel is established as a form of communication to ask questions
+regarding Git usability.
```
Use bullet point format as in template

DaniMori replied:
Did we mention this?

Cristrinaranjus replied:
Yes, because several doubts arose about how to use Git and we referred to the channel to solve them and not take up the time of the meeting session (in excess).

DaniMori replied:
Use "bullet points" is pending

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,98 @@
+The Teams channel is established as a form of communication to ask questions
+regarding Git usability.
+
+# **Task summary:**
```
Fill in task summary with the tasks decided upon in the action points

DaniMori reviewed (changes requested):
Content excellent!

Check out the template and try to stick to the format please!

Cristrinaranjus committed:
Corrections proposed by Dani applied.

Cristrinaranjus committed:
Update 2022-12-02.md

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -5,30 +5,35 @@ output: github_document
 
 **Attendees:** \* Lea Francia \* Roberto Mediavilla \* Daniel Morillo \*
-Cristina Rodríguez \* Jesús Sierralaya
+Cristina Rodríguez \* Jesus Sierralaya
```
Separate names by commas

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -5,30 +5,35 @@ output: github_document
 # Action points
 
-#### Debate
-
```
Great!!

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -49,15 +54,14 @@ For example, in the case of Cris it would be `template-fillin/reusability`
     to github usability issues.
 
--   **Rejected**:
+-   **Rejected**: None.
```
Delete this entry

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -49,15 +54,14 @@ For example, in the case of Cris it would be `template-fillin/reusability`
-    -   It would be necessary to create a branch that comes out of the branch
-        `template-fillin` and that corresponds to our section. For example, in
-        the case of Cris it would be `template-fillin/reusability`
+    -   bifurcaste branch `drafting/template-fillin` to fill in the corresponding 
```
"bifurcate"

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -49,15 +54,14 @@ For example, in the case of Cris it would be `template-fillin/reusability`
     -   Dani, Lea, Jesús and Cris will review will review a part of the skeleton
```
These bullet points are already in tasks, so they are redundant here I think.

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -49,15 +54,14 @@ For example, in the case of Cris it would be `template-fillin/reusability`
+    -   bifurcaste branch `drafting/template-fillin` to fill in the corresponding 
+        section; e.g., `drafting/reusability` to fill in section "Reusability"
 
 -   **Postponed**: *Approve of the previous minutes*. We left pending for the
```
Only "Approve of the previous minutes" is necessary, the rest is "literature" ;)

DaniMori replied:
Mind also that this should have its own action point (it's not related to "Skeleton of the proposal")

DaniMori reviewed (changes requested):
There are just a few minor suggestions to improve it. Well done up to now, everything very clear!

Cristrinaranjus committed:
Update 2022-12-02.md

I have now understood the logic of the headings you proposed, Dani. I have also removed narrativity from the subject. I hope there are no more changes to be made :)

DaniMori committed:
Automatically applies "canonical mode" by turning on the "Visual mode"

DaniMori committed:
Indents bullet points to "sub-level"

DaniMori committed:
Formats "Agreements" in "Skeleton of the proposal" according to template

DaniMori committed:
Deletes redundant text (already in "Agreements")

DaniMori committed:
Formats "Agreements" according to template

DaniMori reviewed (commented):
@Cristrinaranjus I have added a few commits implementing minor changes; please check them out

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,101 @@
+## Jesus
+
+-   Fill in field "Follow-up"
+-   Upload minutes of previous meeting
```
This task is missing in the "Tasks" subsection in action point "Approval of minutes..."

DaniMori commented on doc/minutes/2022-12-02.md:
```diff
@@ -0,0 +1,101 @@
+output: github_document
+---
+
+**Attendees:** \* Lea Francia, \* Roberto Mediavilla, \* Daniel Morillo, \*
```
Leave only commas (no "*" symbols)

DaniMori reviewed (changes requested):
Two minor changes: format (the "stars"), and a missing task in the "Action points" ;)

Cristrinaranjus committed:
Update 2022-12-02.md

DaniMori reviewed (dismissed):
Well done!!

robertomediavilla committed:
creates html file (check if that was a mistake)

DaniMori committed:
Revert "creates html file (check if that was a mistake)"

This reverts commit 7aba8d96f6beef0c95fa624b1dfc7de5e2bc6f0c.

DaniMori reviewed (commented):
(Minor) format change only (please!) :)

Cristrinaranjus committed:
Update 2022-12-02.md

The use of bullet points it's no longer pending.

Cristrinaranjus commented:
@JesusSierralaya could you please review these changes and approve if you consider it already finished? Thanks!
